test/Microsoft.AspNetCore.Server.KestrelTests/KestrelServerTests.cs

```diff
@@ -23,7 +23,7 @@ public class KestrelServerTests
         [InlineData(-1337)]
         public void StartWithNonPositiveThreadCountThrows(int threadCount)
         {
-            var testLogger = new TestApplicationErrorLogger();
+            var testLogger = new TestApplicationErrorLogger { PrintCriticalErrors = false };
```

Comment:
I want to see critical errors in the test output only when they are unexpected. Arguably we should actually change this to ThrowOnCriticalErrors so we notice a test failure. Especially now that the console test output is much noisier by default.